### ISSUE TITLE
Support nfs-utils version ~2.2.1

### ISSUE THREAD

**Issue by kovacskri** [spryker/devvm#48]
nfs-utils disabled serving NFS over UDP in version 2.2.1. Trying to run the spryker box with it gives you the following error message:

`mount.nfs: requested NFS version or transport protocol is not supported`

To solve it you need to explicitly specify the synced folders in your Vagrantfile, and disable udp for each of them.

```
config.vm.synced_folder ".",   "/vagrant" , nfs_udp: false
config.vm.synced_folder "project", "/data/shop/development/current", nfs_udp: false
```

**Comment by marek-obuchowicz** [comment]
Are we able to detect it somehow and make sure that we can apply this setting automatically? I checked release notes of nfs-utils 2.1.1 and they don't include anything about removing UDP protocol - actually one of the features in this version improves one of UDP-specific features.

I'm wondering if it might be relted not to nfs-utils itself, but rather to its configuration - or defaults provided by distribution. Can you please let me know which package version and distribution are you using?

If we can't find a proper method to detect it, we might add a configurable option via env variable to force TCP + document known use cases (ie. which distributions or package versions are affected).

**Comment by kovacskri** [comment]
Wouldn't it be a solution to always disallow UDP if the host system is Linux?

I am using Arch Linux with the following packages installed:
Vagrant 2.0.2
Virtualbox 5.2.6r120293
nfs-utils-2.3.1-1

Kernel: x86_64 Linux 4.15.1-2-ARCH

I am referring to a known problem in the Arch Wiki.
https://wiki.archlinux.org/index.php/NFS/Troubleshooting#Problems_with_Vagrant_and_synced_folders

**Comment by marek-obuchowicz** [comment]
This feature (use TCP by default) will be released in the upcoming dev vm. The release is now being tested internally and will be available soon as public.

**Comment by marek-obuchowicz** [comment]
It has been released in version `v2.1.0` 👍
